libs/Microsoft.MixedReality.WebRTC.Unity/Assets/Microsoft.MixedReality.WebRTC.Unity/Scripts/Media/RemoteAudioSource.cs

```diff
@@ -40,6 +40,163 @@ public class RemoteAudioSource : AudioSource
         /// </summary>
         private ConcurrentQueue<Action> _mainThreadWorkQueue = new ConcurrentQueue<Action>();
 
+        /// <summary>
+        /// Clip corresponding to the remote audio track.
+        /// </summary>
+        private AudioClip _audioClip; // created when the first webrtc frame arrives in RemoteAudioFrameReady.
+
+        /// <summary>
+        /// Set when we've enqueued the job to create the audio clip on the main thread. (to avoid duplication)
+        /// </summary>
+        private volatile bool _audioClipEnqueuedCreate = false;
+
+        /// <summary>
+        /// Internal adaptor from webrtc frame data to unity audio clip.
+        /// </summary>
+        class AudioData
+        {
+            float[] _buffer = null;
+            bool _isFull = false;
+            int _readPos = 0; // AudioClip fetches start from here
+            int _writePos = 0; // WebRTC data gets appended here
+            int _channelCount = 0;
+            int _sampleRate = 0;
```

Comment:
Use an explicit `private` keyword please by convention, for clarity. And same below.